### PR TITLE
Add the active selection end.

### PR DESCRIPTION
**Pull request by sean-mcmanus**
This requires the engine changes or it'll crash (hopefully, no externals will be using master?).
Don't check this in until the engine part is done.

### CODE REVIEW

**Comment by sean-mcmanus** [comment]
This change isn't ready yet...it's not sending the expected JSON. VS Code is sending the JSON as 2 unnamed objects with line/character (i.e. 2 positions) -- I think I can just parse the json as 2 positions instead of a Range (which doesn't work because there are no "start/end" objects being sent in the JSON).

File: Extension/package.json [hunk 1]
```diff
@@ -63,7 +63,7 @@
     ],
     "configuration": {
       "type": "object",
-      "title": "C/C++ Configuration",
```
Comment (bobbrow) [inline Extension/package.json:66]: what's this about?

**Review by bobbrow — APPROVED** [review]

File: Extension/package.json (hunk 1 above)
Comment (sean-mcmanus) [reply Extension/package.json:66]: It matches the other extensions in the new VS Code settings view. I can show you.

Comment (bobbrow) [reply Extension/package.json:66]: ok

**Pull request merged**